WordPress/gutenberg
# Make usage of the search icon more consistent and meaningful

issue: https://github.com/WordPress/gutenberg/issues/63023

## What?
Changed icons to `menu` for the command part and `plus` for the Zoom In functionality in the Image block toolbar.

## Why?
This PR updates the icons used in the Gutenberg editor to improve visual consistency and clarity. The `menu` icon is now used for the command part, aligning with standard UI practices for menu interactions. The `plus` icon replaces the previous icon for Zoom In in the Image block toolbar, making it more intuitive and recognizable for users.

## How?
- Updated the `ZoomDropdown` component in `components/image-editor/zoom-dropdown.js` to use the `plus` icon for Zoom.
- Modified the render functions of the `Dropdown` component to conditionally render the `menu` and `plus` icons based on specific states.
- Imported and utilized the icons from the `@wordpress/icons` library.

## Testing Instructions
1. Open a post or page in the Gutenberg editor.
2. Insert an Image block.
3. Verify that the toolbar displays a `plus` icon for Zoom In in the Image block.
4. Check that the command part of the toolbar now shows a `menu` icon.
5. Perform Zoom In and Zoom Out actions to ensure the functionality works correctly with the new icons.

### Testing Instructions for Keyboard
- Navigate to an Image block using keyboard navigation.
- Ensure that the Zoom In functionality can be accessed and used effectively using keyboard shortcuts or tab navigation.
- Verify that the command part of the toolbar is accessible and labeled correctly for screen readers.

## Screenshots or screencast 
Screenshots or a screencast demonstrating the updated icons in action would be added here.
![image](https://github.com/WordPress/gutenberg/assets/90276347/e4193df5-42c3-4fcc-ab7e-a576dbeadb57)
